src/Microsoft.Graph.Cli.Core/IO/AuthenticationCacheUtility.cs

```diff
@@ -121,6 +194,23 @@ private async Task WriteConfigurationAsync(string path, ConfigurationRoot config
         }
     }
 
+    private async Task<MsalCacheHelper> GetProtectedCacheHelperAsync(string name)
+    {
+        // https://github.com/Azure/azure-sdk-for-net/blob/4b2579556b7271587d2fb122163e23090a043597/sdk/identity/Azure.Identity/src/Constants.cs
+        string cacheDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), ".IdentityService");
+        const string keychainService = "Microsoft.Developer.IdentityService";
```

Comment:
could we get some of those values with reflection instead?

Reply:
Since these APIs are internal, refactors of the variables will lead to runtime errors. Isn't that risky?

Reply:
hence the suggestion, at least with references like nameof and typeof we'll get a build break, earlier feedback than runtime error.

Reply:
The constants that hold these values are internal to the Azure.Identity assembly.

Reply:
sure, but couldn't we grab a type in MSAL to have some symbol binding instead of a string constant?

Reply:
`Microsoft.Developer.IdentityService` isn't a type.

Reply:
thanks for clarifying that aspect, resolving